### DISCUSSION TITLE
ShutterButton Advanced Control

### DISCUSSION THREAD

**Discussion by caprez95** [arendst/Tasmota#19569] [Q&A]
Hello together

I have some problems with the Shutter Buttons.
I defined two Buttons for 1 Shutter as follow:

ShutterButton1 1 - - - 100 - - - - -
ShutterButton1 2 - - - 0 - - - - -

That means:
MultiPress (Single, Double and Triple-Press) deactivated and only ButtonHold active

<img width="394" alt="ShutterButton_Advanced" src="https://github.com/arendst/Tasmota/assets/45364281/1ccbfddf-7725-4191-8d98-95b4ef84df3b">

Thats works fine. MultiPress (up to triple-press) is ignored and only on Button Hold the Shutter moves, and on Button release the Shutter Stopps.

BUT, unfortunately the ShutterButton reacts to a Quadruple-Press.
After a Quadruple-Press, the relay remains on permanently until the upper or lower end position of the shutter is reached (see video).

https://github.com/arendst/Tasmota/assets/45364281/ba5f95cd-fca5-4578-950b-6857e72b3734

But the quadruple-Press cannot be defined for a shutter button.

Does anyone know where this comes from?
Does this come from the button multi-press functions? 
Are they active on the shutter buttons? 
And can the multi-press functions be deactivated?

Many thanks for every answer.
Kind regards Carlo

**Comment by jtauscher** [comment]
I'm not sure but I think quadruple press activates some group topic command.
